src/validator/amazonLink.validator.ts

```diff
@@ -0,0 +1,18 @@
+import { ExpressValidator } from 'express-validator';
```

Comment:
move into validations class please

Reply:
Maybe extract validations for each route in its own file then to keep everything in one place?

Reply:
we can do that too, sure

I'd suggest putting them into a folder (validations or something like that) inside middleware and naming the files according to the route

It probably needs some refactoring to have a base class for error handling and the validate function since this is used in all validation classes and extend all other validation classes from this one

Reply:
Okay, il move this for now and we should create a separate ticket for  this.